Client/Frontend/Autofill/CreditCard/CreditCardSettingsView/CreditCardItemRow.swift

```diff
@@ -78,17 +76,14 @@ struct CreditCardItemRow: View {
             .padding(.trailing, 16)
             .padding(.top, 11)
             .padding(.bottom, 11)
-            .background(isTapped ? backgroundHoverColor : backgroundColor)
-            .gesture(
-                TapGesture()
-                    .onEnded { _ in
-                        didSelectAction?()
-                    }
-                    .simultaneously(with: DragGesture(minimumDistance: 0))
-                    .updating($isTapped) { (_, isTapped, _) in
-                        isTapped = true
-                    }
-            )
+            .background(isTapping ? backgroundHoverColor : backgroundColor)
+            .onTapGesture {
+                isTapping = true
+                DispatchQueue.main.asyncAfter(deadline: .now() + 0.1) {
+                    isTapping = false
```

Comment:
```suggestion
```
See my explanation below for the removal of isTapping = false

https://github.com/mozilla-mobile/firefox-ios/pull/15335/files?diff=unified&w=0#r1255165876